### PR TITLE
DOCSP-33403 Correct Prettify Code Shortcut

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## DESCRIPTION

Correction to Prettify shortcut key in Compass and a few other corrections to Mac commands.

## STAGING

https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-33403/keyboard-shortcuts/

## JIRA

https://jira.mongodb.org/browse/DOCSP-33403

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65171632492df295cf115f27

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by mdb-ashley — APPROVED** [review]
LGTM! Thanks for the quick pickup, Ian.

**Comment by ianf-mongodb** [comment]
@mmarcon Would you mind taking a look at this?

**Review by mmarcon — APPROVED** [review]

File: source/keyboard-shortcuts.txt [hunk 1]
```diff
@@ -194,35 +194,35 @@ Keyboard shortcuts enable you to easily navigate |compass-short|.
...
+     - Swap the last two characters before the cursor
+     - ``Ctrl`` + ``T``
      - ``Ctrl`` + ``T``
-     - ``Cmd`` + ``T``
 
    * - Web Shell
-     - Changes the line to uppercase.
+     - Changes the line to uppercase
      - ``Ctrl`` + ``U``
      - ``Cmd`` + ``U``
```
Comment (mmarcon) [inline source/keyboard-shortcuts.txt:227]: This actually does not seem to work. Maybe let's drop it for now.

**Review by mmarcon — CHANGES_REQUESTED** [review]

File: source/keyboard-shortcuts.txt (hunk 1 above)
Comment (ianf-mongodb) [reply source/keyboard-shortcuts.txt]: Removed.

**Review by jaoppenheim — APPROVED** [review]

**Pull request merged**